homeassistant/components/comelit/coordinator.py

```diff
@@ -26,7 +26,20 @@
 type ComelitConfigEntry = ConfigEntry[ComelitBaseCoordinator]
 
 
-class ComelitBaseCoordinator(DataUpdateCoordinator[dict[str, Any]]):
+class AlarmDataObject(TypedDict):
```

Comment:
In the future, the library should preferably use this typed dict or even better use a data class for the return value from the library. Then we don't need to define it here and cast the return value to this type.

Reply:
https://github.com/chemelli74/aiocomelit/pull/242